web/impact/impact/tests/test_office_hours_calendar_view.py

```diff
@@ -44,45 +46,69 @@
 class TestOfficeHoursCalendarView(APITestCase):
     view = OfficeHoursCalendarView
 
-    def test_no_date_specified_sees_current_week(self):
+    def test_no_focal_dateified_sees_current_week(self):
         office_hour = self.create_office_hour()
         response = self.get_response(user=office_hour.mentor)
         self.assert_hour_in_response(response, office_hour)
 
-    def test_no_date_specified_does_not_see_last_week(self):
+    def test_no_focal_dateified_does_not_see_last_week(self):
         office_hour = self.create_office_hour(
             start_date_time=days_from_now(-9))
         response = self.get_response(user=office_hour.mentor)
         self.assert_hour_not_in_response(response, office_hour)
 
-    def test_date_specified_sees_sessions_in_range(self):
+    def test_focal_dateified_sees_sessions_in_range(self):
         two_weeks_ago = days_from_now(-14)
-        date_spec = two_weeks_ago.strftime(ISO_8601_DATE_FORMAT)
+        focal_date = two_weeks_ago.strftime(ISO_8601_DATE_FORMAT)
         office_hour = self.create_office_hour(
             start_date_time=two_weeks_ago)
         response = self.get_response(user=office_hour.mentor,
-                                     date_spec=date_spec)
+                                     focal_date=focal_date)
         self.assert_hour_in_response(response, office_hour)
 
-    def test_date_specified_does_not_see_sessions_not_in_range(self):
+    def test_calendar_data_for_month_span_last_day_of_the_month(self):
+        today = datetime.now()
+        days_in_month = calendar.monthrange(today.year, today.month)[1]
```

Comment:
Tests should be consistent, ie, they should test the same thing every time. Please make this test a consistent month. By convention, we tend to use a month in the past. 
You can use freezegun to force the time to be some particular datetime (see examples in the codebase)